Examples/Io/Root/src/RootTrajectorySummaryWriter.cpp

```diff
@@ -150,6 +195,7 @@ ActsExamples::RootTrajectorySummaryWriter::finalize() {
   m_outputTree->Write();
   m_outputFile->Close();
 
+  ACTS_INFO("Wrote full covariance matrix to tree");
```

Comment:
This probably should only be printed if the option is actually `true`, right?

Reply:
yes, sorry for that.

Reply:
I changed that in https://github.com/owtscharenko/acts/commit/46e8bce2a9b9c9691b3ceec40be4053b04cbb264